.github/workflows/ci.yml

```diff
@@ -20,6 +20,13 @@ jobs:
     uses: ./.github/workflows/build_nilrt.yml
   create_client_artifacts:
     uses: ./.github/workflows/create_client_artifacts.yml
+  create_release:
+    if: ${{ startsWith(github.ref, 'refs/heads/releases') }}
```

Comment:
And what will happen if you have a draft open and this runs again? Will it update the draft or make a new one?

Reply:
If there's a merge or cherry-pick after the release branch is created. It'll run the step again as part of the CI and create a new draft release similar to how it would on a new release branch.

I added to the if statement to avoid the create_release step running on PRs to a releases/* branch  though by adding `&& github.event_name == 'push'`. I'm testing that on a PR from this branch to a releases/* branch in my fork to make sure that it doesn't get kicked off.

It will make a new draft release currently. I could switch it to have an "allowUpdates" on the actual create_release action which I think will replace a draft release with the same proposed tag if it exists. If you look at my release drafts on my fork (https://github.com/reckenro/grpc-device/releases) right now I have a bunch of drafts triggered by different iterations of this action that would create the tag v1.5.2 if they were published.